src/applications/vaos/actions/appointments.js

```diff
@@ -84,10 +85,15 @@ async function getAdditionalFacilityInfo(futureAppointments) {
   // Get facility ids from non-VA appts or requests
   const nonVaFacilityAppointmentIds = futureAppointments
     .filter(
-      appt => appt.vaos?.videoType || appt.vaos?.isCommunityCare || !appt.vaos,
```

Comment:
This was keeping us from downloading facility info for video appointments, which we need for the cancel modal